src/officehours_api/consumers.py

```diff
@@ -289,6 +299,9 @@ def send_user_deleted(user_id: int, channel_layer=None):
 @receiver(post_save, sender=User)
 def trigger_user_update(sender, instance: User, **kwargs):
     transaction.on_commit(lambda: send_user_update(instance.id))
+    # Tried to also get hostnames to update on change in the edit queue page, but this didn't do it alone
+    # for queue in list(instance.queue_set.all()):
+    #     transaction.on_commit(lambda: send_queue_update(queue.id))
```

Comment:
To make that work, it looks like we'd have to trigger updates to `QueueConsumer`s here. Alternatively, we could subscribe individually to every user websocket for which a page has information (costly). I don't think the benefit of doing it would be worth the cost of either fix, as users are essentially never updated, and even if they were, the updates wouldn't need to be seen immediately.

Reply:
I agree, probably not worth it. I can just forget about that. Seems like we may want to handle a completely deleted user? Going to keep looking at these signal receivers.

Reply:
So after looking at these receivers more, I think it's okay for now? I played around with getting hosts to disappear from edit queue pages when they are deleted, but that's actually kind of out-of-scope for this PR, and maybe not that important for some of the same reasons you outlined above? I think updates via WebSocket to "Manage Queues" when a `Host`-to-`Queue` relation is added or deleted seems sufficient for this issue, and that all should be working.